src/he/index.ts

```diff
@@ -0,0 +1,64 @@
+const { DesiloSealFactory } = require('./desilo');
+import { CipherText } from 'node-seal/implementation/cipher-text';
+
+const PARAMS = {
+  polyModulusDegree: 8192,
+  coeffModulusArray: Int32Array.from([60, 40, 40, 60]),
+  scaleBit: 40
+};
+export default class HomomorphicEncryption {
+  public seal: any;
+  private _initialized: boolean;
+
+  constructor() {
+    this.seal = null;
+    this._initialized = false;
+  }
+
+  async init(keys?: any) {
```

Comment:
`params` for init parameter? (use current PARAMS value as default)

Reply:
Good idea! Done, thanks.